### PR TITLE
Added new icons for "To Agree" and "Info" (with outer circle)

### PR DESCRIPTION
**Pull request by EPENNACHINI**
## Motivation
Se agregan nuevos iconos:
- To Agree, o "Acuerdos" (16, 20, 24, 32, 48)
- Info, con la circunferencia fuera de la **`i`** (16, 20, 24)

Ejemplos
| **To Agree** | **Info** |
| ----- | ------ |
| andes_ui_to_agree_20 | andes_ui_info_20 |

## Description
Se agregan los íconos antes mencionados ya que se encuentran en [carpeta de Drive](https://drive.google.com/drive/folders/1aN9bPS_kshCyEs9ep-ExBUinkUwLThGr) de Andes pero no como recurso en el proyecto. Adjunto el correspondiente [PR de iOS](https://github.com/mercadolibre/fury_andesui-ios/pull/209).

## Affected component
`N/A`

## Frontify component link
`N/A`

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [ ] `[N/A]` I have coded an example inside the Demo App,
- [ ] `[N/A]` I have added proper tests,
- [x] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] `[N/A]` I have updated GitHub wiki,
- [x] I have the explicit approval of one or more members of the UX Team,
- [ ] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [ ] This code adds a new component
- [ ] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Review by cvillamilmeli — APPROVED** [review]

**Comment by eportasML** [comment]
@EPENNACHINI las convertis a webp? Todas los drawables de andes estan en esa extension. Deberia pesar un poco menos
Captura de Pantalla 2021-01-26 a la(s) 11 22 44

**Comment by EPENNACHINI** [comment]
> @EPENNACHINI las convertis a webp? Todas los drawables de andes estan en esa extension. Deberia pesar un poco menos
> Captura de Pantalla 2021-01-26 a la(s) 11 22 44

Perdón @eportasML  se me pasó, ya lo hago y lo pusheo.

**Comment by EPENNACHINI** [comment]
@eportasML ya convertí las imágenes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Pending definition
```
Comment (eportasML) [inline CHANGELOG.md:1]: Cambialo a 3.7.0 y lo mismo en el libraryVersion de gradle.properties

Comment (EPENNACHINI) [reply CHANGELOG.md:1]: listo, dale, ya lo subo

Comment (EPENNACHINI) [reply CHANGELOG.md]: hecho

**Review by eportasML — APPROVED** [review]

**Pull request merged**